src/main/java/com/commercetools/pspadapter/notification/processor/impl/NotificationProcessorBase.java

```diff
@@ -40,7 +40,7 @@
         this.gson = gson;
     }
 
-    abstract List<? extends UpdateAction<Payment>>  createChangeTransactionState(@Nonnull Payment ctpPayment);
+    abstract List<? extends UpdateAction<Payment>> updateCtpTransactions(@Nonnull Payment ctpPayment, @Nonnull Event event);
```

Comment:
the new name is also confusing: actually you don't update, but build update actions list. Pls, change the function name so it reflects actual behavior.

Reply:
https://github.com/commercetools/commercetools-paypal-plus-integration/pull/64/commits/f081fc062561e229c4a3e1464ed6bc82c353b3b2